app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -14,3 +14,5 @@
 <%= select_f f, :keyboard_layout, f.object.keyboard_layouts,:downcase, :to_s, { }, :label => _("Default VNC Keyboard") %>
 <%= textarea_f f, :public_key, :label => _("X509 Certification Authorities"), :size => "col-md-8",
   :placeholder => _("Optionally provide a CA, or a correctly ordered CA chain. If left blank, a self-signed CA will be populated automatically by the server during the first request.") %>
+<%= alert :header => '', :class => 'alert-warning', :close => false,
+          :text => _("API v3 is deprecated and will be removed completely in foreman 2.4") %>
```

Comment:
Should this be next to the api version checkbox? perhaps the help text for it should also be changed?